modules/combined/test/tests/beam_eigenstrain_transfer/master_uo_transfer.i

```diff
@@ -0,0 +1,233 @@
+    variable = rot_x
+    boundary = 1
+    value = 0.0
+  [../]
+  [./fixr2]
+    type = DirichletBC
+    variable = rot_y
+    boundary = 1
+    value = 0.0
+  [../]
+  [./fixr3]
+    type = DirichletBC
+    variable = rot_z
+    boundary = 1
+    value = 0.0
+  [../]
+[]
+
+[Kernels]
+  [./solid_disp_x]
+    type = StressDivergenceBeam
+    block = '1'
+    displacements = 'disp_x disp_y disp_z'
+    rotations = 'rot_x rot_y rot_z'
+    component = 0
+    variable = disp_x
+  [../]
+  [./solid_disp_y]
+    type = StressDivergenceBeam
+    block = '1'
+    displacements = 'disp_x disp_y disp_z'
+    rotations = 'rot_x rot_y rot_z'
+    component = 1
+    variable = disp_y
+  [../]
+  [./solid_disp_z]
+    type = StressDivergenceBeam
+    block = '1'
+    displacements = 'disp_x disp_y disp_z'
+    rotations = 'rot_x rot_y rot_z'
+    component = 2
+    variable = disp_z
+  [../]
+  [./solid_rot_x]
+    type = StressDivergenceBeam
+    block = '1'
+    displacements = 'disp_x disp_y disp_z'
+    rotations = 'rot_x rot_y rot_z'
+    component = 3
+    variable = rot_x
+  [../]
+  [./solid_rot_y]
+    type = StressDivergenceBeam
+    block = '1'
+    displacements = 'disp_x disp_y disp_z'
+    rotations = 'rot_x rot_y rot_z'
+    component = 4
+    variable = rot_y
+  [../]
+  [./solid_rot_z]
+    type = StressDivergenceBeam
+    block = '1'
+    displacements = 'disp_x disp_y disp_z'
+    rotations = 'rot_x rot_y rot_z'
+    component = 5
+    variable = rot_z
+  [../]
+[]
+
+[Materials]
+  [./elasticity]
+    type = ComputeElasticityBeam
+    youngs_modulus = 1e9
+    poissons_ratio = 0.3
+    shear_coefficient = 1.0
+    block = 1
+  [../]
+  [./strain]
+    type = ComputeIncrementalBeamStrain
+    block = '1'
+    displacements = 'disp_x disp_y disp_z'
+    rotations = 'rot_x rot_y rot_z'
+    area = 0.5
+    Ay = 0.0
+    Az = 0.0
+    Iy = 0.01
+    Iz = 0.01
+    y_orientation = '0.0 0.0 1.0'
+    eigenstrain_names = 'thermal'
+  [../]
+  [./stress]
+    type = ComputeBeamResultants
+    block = 1
+  [../]
+  [./thermal]
+    type = ComputeEigenstrainBeamFromVariable
+    displacement_eigenstrain_variables = 'zero1 to_var zero2'
+    eigenstrain_name = thermal
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+
+  #Preconditioned JFNK (default)
+  solve_type = 'PJFNK'
+
+  line_search = 'none'
+
+  l_max_its = 50
+  nl_max_its = 50
+  nl_rel_tol = 1e-12
+  nl_abs_tol = 1e-12
+  l_tol = 1e-9
+
+  start_time = 0.0
+  end_time = 0.075
+  dt = 0.0125
+  dtmin = 0.0001
+[]
+
+[AuxVariables]
+  [./to_var]
+  [../]
+  [./zero1]
+  [../]
+  [./zero2]
+  [../]
+[]
+
+[MultiApps]
+  [./sub]
+    type = TransientMultiApp
+    app_type = CombinedApp
+    positions = '1.5 2.0 2.0 2.5 0.0 3.0'
+    input_files = 'subapp1_uo_transfer.i subapp2_uo_transfer.i'
+  [../]
+[]
+
+[Transfers]
+  [./fromsub]
+    type = MultiAppUserObjectTransfer
+    direction = from_multiapp
+    user_object = axial_str
+    multi_app = sub
+    variable = to_var
```

Comment:
Where it makes sense, let's use that injective option in these tests.

Reply:
I have made this change.